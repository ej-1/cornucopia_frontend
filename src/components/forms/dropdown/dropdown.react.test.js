// Link.react.test.js
import React from "react";
import renderer from "react-test-renderer";
import DropDown from "../dropdown/dropdown";

test("Footer gets rendered", () => {
  const handleChange = () => {};

  const component = renderer.create(
    <DropDown
      title="the nice title"
      onSelect={handleChange} // MOCK IT.
      options={["OPTION NUMBER 1", "OPTION NUMBER 2", "OPTION NUMBER 3"]}
    />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
