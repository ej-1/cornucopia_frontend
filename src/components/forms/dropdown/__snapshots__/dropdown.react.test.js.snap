// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer gets rendered 1`] = `
<div
  className="dropdown btn-group"
>
  <button
    aria-expanded={false}
    aria-haspopup={true}
    className="dropdown dropdown-toggle btn btn-default"
    disabled={false}
    id="dropdown-the nice title"
    onClick={[Function]}
    onKeyDown={[Function]}
    role="button"
    type="button"
  >
    Choose the nice title
     
    <span
      className="caret"
    />
  </button>
  <ul
    aria-labelledby="dropdown-the nice title"
    className="dropdown-menu"
    role="menu"
  >
    <li
      className=""
      role="presentation"
      style={undefined}
    >
      <a
        href="#"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="menuitem"
        tabIndex="-1"
      >
        OPTION NUMBER 1
      </a>
    </li>
    <li
      className=""
      role="presentation"
      style={undefined}
    >
      <a
        href="#"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="menuitem"
        tabIndex="-1"
      >
        OPTION NUMBER 2
      </a>
    </li>
    <li
      className=""
      role="presentation"
      style={undefined}
    >
      <a
        href="#"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="menuitem"
        tabIndex="-1"
      >
        OPTION NUMBER 3
      </a>
    </li>
  </ul>
</div>
`;
